client/state/sites/selectors.js

```diff
@@ -134,6 +134,23 @@ export function isJetpackSite( state, siteId ) {
 }
 
 /**
+ * Returns true if site is a Pressable Jetpack site, false if the site is hosted on
+ * WordPress.com or is a regular Jetpack, or null if the site is unknown.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is a Pressable site
+ */
+export function isPressableSite( state, siteId ) {
+	const site = getRawSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+
+	return site.options.pressable;
```

Comment:
👍 Thanks for adding this. Let's check in with @peterbutler as work starts on this.